app/views/layouts/publishers.html.slim

```diff
@@ -2,4 +2,9 @@
   .main-content
     #content.container
       = yield
-= render(template: "layouts/application")
+
+= render \
+  template: "layouts/application",
+  locals: { \
+    viewport_initial_scale: ((action_name === 'new_auth_token') && '1.0') \
```

Comment:
This limits these changes to the login page, and ensures they do not impact the dashboard itself.